tests/test_network.py

```diff
@@ -92,6 +92,28 @@ def test_list_nodes(network, session):
     assert set(network_node_ids).issubset(set(node_ids))
 
 
+def test_add_compute_plan(factory, session):
```

Comment:
Currently all tests executing stuff (and then "quite long" tests) are located inside the `test_execution*` files.

I think we should keep it that way to avoid having too many long tests. I think we should simply use and update an existing test. We could maybe rename this tests file to make things clearer or we could discuss about how organizing the tests.

Reply:
In this specific case, we're just testing that the create and get return the same values and that all sub-items were created. There are other tests that check that the execution went well, it's just not the purpose here.